flutter_local_notifications/lib/src/platform_specifics/android/status_bar_notification.dart

```diff
@@ -0,0 +1,22 @@
+class ActiveNotification {
```

Comment:
Can you rename the file to match the name of the class?

Reply:
sure